src/tls/Makefile

```diff
@@ -20,7 +22,9 @@ STDLIB=FStar.Ghost.fst FStar.FunctionalExtensionality.fst FStar.Classical.fst FS
 stHyperHeap.fst allHyperHeap.fst FStar.Monotonic.RRef.fst char.fsi FStar.String.fst FStar.List.fst FStar.ListProperties.fst \
 FStar.Seq.fst FStar.SeqProperties.fst #FStar.Int64.fst 
 CODEGEN_LIBS=CoreCrypto Platform Classical SeqProperties HyperHeap
-FSTAR=$(FSTAR_HOME)/bin/fstar.exe $(STDLIB) $(addprefix --codegen-lib , $(CODEGEN_LIBS)) $(OTHERFLAGS)
+
+FSTARFLAGS=$(call add_stdlib_prefix,$(STDLIB)) $(addprefix --codegen-lib , $(CODEGEN_LIBS)) $(OTHERFLAGS)
```

Comment:
NuBuild needs to be able to find these files too and there was (and still is, afaik) no way to ask F* which include directory a given file in the argument list was found.

I'm really unhappy with this approach. I'm in the process of changing NuBuild so that it builds its own search path. That way, there is no need to specify relative paths to files listed on the F* command line.